### DISCUSSION TITLE
Script to determine segment inclusion besides include_folders/exclude_folders

### DISCUSSION THREAD

**Discussion by piejanssens** [JanDeDobbeleer/oh-my-posh#1848] [Ideas]
I'm sure there are many, but to give you one use case: Python virtual environments
I don't want a Python segment present all the time. 
I only want it in folders **containing** python files or folder that **contain** a `something/pyvenv.cfg` file.

```
{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  ...
  "blocks": [
    {
      ...
      "segments": [
        {
          "type": "path",
          "style": "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground": "#ffffff",
          "background": "#61AFEF",
          "properties": {
          	"include_check" : "$ZDOTDIR/oh-my-posh/check_pyenv"
            ...
          }
        }
      ]
    }
  ]
}
```

By convention oh-my-posh could supply the current working directory to the script and expect a true or false string.
The script could be any terminal executable/script in a language supported by the host system which is capable of echoing a string.

**Comment by JanDeDobbeleer** [comment]
Why don't you set the `display_mode` to `files` in this case?

**Reply by piejanssens** [reply]
Thanks, that solves my need for Python/venv!
